frontend/src/store/configurate-episode/actions.ts

```diff
@@ -7,14 +7,13 @@ import {
 } from 'common/types/types';
 import { getFileFromFileList, getDataUrl } from 'helpers/helpers';
 import { ActionType } from './common';
-import { DEFAULT_PODCAST_ID } from 'common/constants/constants';
 import { NotificationMessage, NotificationTitle } from 'common/enums/enums';
 
 const createEpisode = createAsyncThunk<Episode, EpisodeFormPayload, AsyncThunkConfig>(
   ActionType.CREATE_EPISODE,
   async (createEpisodePayload, { getState, extra }) => {
     const { episodeApi, notificationService } = extra;
-    const { auth } = getState();
+    const { auth, podcast } = getState();
```

Comment:
```suggestion
    const { auth } = getState();
```
what if you go directly on the edit podcast page?
we need to take `podcastId` from router and pass it to this action